.github/workflows/performance.yml

```
@@ -65,9 +65,19 @@ jobs:
   # - Install MU plugin.
   # - Run performance tests (current commit).
   # - Print performance tests results.
+  # - Check out target commit (target branch or previous commit).
+  # - Install npm dependencies.
+  # - Build WordPress.
+  # - Run performance tests (previous/target commit).
+  # - Print target performance tests results.
+  # - Reset to original commit.
   # - Set the environment to the baseline version.
   # - Run baseline performance tests.
-  # - Print base line performance tests results.
+  # - Print baseline performance tests results.
+  # - Compare results with base.
+  # - Add workflow summary
+  # - Check if a comment was already made
+  # - Comment on PR with test results
```

```suggestion
  # - Add workflow summary.
  # - Check if a comment was already made.
  # - Comment on PR with test results.
```

Add missing end period.